prime-dotnet-webapi/Models/BusinessEvent.cs

```diff
@@ -50,5 +50,7 @@ public string AdminIDIR
         public string Description { get; set; }
 
         public DateTimeOffset? EventDate { get; set; }
+
+        public string PartyName { get; set; }
```

Comment:
On business event we already have the party as a relation, so this could either be a computed column on this model, or when we map the view model for these events we can map party name when we project.

We do not want to store this name on this event.

Reply:
This may be a great time to make view models for business events since we only want party name on non enrollee business events. As well we can get rid of all the [JsonIgnores].

Reply:
I did not see @james-hollinger floating comment about this same topic, but yes I concur haha

Reply:
SiteBusinessEventVM created. I didn't delete the JsonIgnores as I think they are still being used on the enrollee business events